salesagility/SuiteCRM
# Fixed #6720 - Remove old recaptcha files

## Description



Removes unused reCaptcha lib.

## Motivation and Context

Issue reference: #6720 

## How To Test This

See https://github.com/salesagility/SuiteCRM/pull/5531 where reCaptcha was moved to composer.

## Types of changes


### Final checklist
